TRENZ.Docs.API.Test/LocalSourceTest.cs

```diff
@@ -70,4 +70,21 @@ public void TestToString(Dictionary<string, string> config, string expected)
         var source = new LocalSource(configuration);
         Assert.AreEqual(expected, source.ToString());
     }
+
+    public static IEnumerable<object[]> FindFilesValuesProvider()
+    {
+        yield return new object[] { new Dictionary<string, string> { { "Type", "local" }, { "Name", "Test" }, { "Root", "./Data/" } }, new[] { "Image", "Test" } };
```

Comment:
Hmm,

* the test `TestFindFiles` would be more useful with some additional edge cases. (For example, test if it finds a `.order` file.)
* I would also prefer if the items here are vertical so that it's easier to see that you're creating a hierarchy.